js/comment-btn-handler.js

```diff
@@ -0,0 +1,25 @@
+export default () => {
+    const commentButton = jQuery('#comment-button');
+    const input = jQuery('#comment-input');
+
```

Comment:
Du hade, om du velat göra det lite snyggare, lägga `input.val()` i en variabel.
```suggestion
const inputVal = input.val();
```